src/main/io/ledstrip.c

```diff
@@ -484,7 +488,19 @@ static void applyLedFixedLayers()
         }
 
         if (ledGetOverlayBit(ledConfig, LED_OVERLAY_THROTTLE)) {
-            hOffset += scaledAux;
+			if (auxInput < (PWM_RANGE_MIN + PWM_RANGE_MAX)/2) //if below average
+				{
+					color.h = scaleRange(auxInput, PWM_RANGE_MIN, (PWM_RANGE_MIN + PWM_RANGE_MAX) / 2, color0.h, color.h);  //Fade the color smoothly from color to the next color from min to max throttle
+					color.s = scaleRange(auxInput, PWM_RANGE_MIN, (PWM_RANGE_MIN + PWM_RANGE_MAX) / 2, color0.s, color.s);
+					color.v = scaleRange(auxInput, PWM_RANGE_MIN, (PWM_RANGE_MIN + PWM_RANGE_MAX) / 2, color0.v, color.v);
+				}
+			else 
+				{
+					color.h = scaleRange(auxInput, (PWM_RANGE_MAX + PWM_RANGE_MIN) / 2, PWM_RANGE_MAX, color.h, color2.h);  //Fade the color smoothly from color to the next color from min to max throttle
+					color.s = scaleRange(auxInput, (PWM_RANGE_MAX + PWM_RANGE_MIN) / 2, PWM_RANGE_MAX, color.s, color2.s);
+					color.v = scaleRange(auxInput, (PWM_RANGE_MAX + PWM_RANGE_MIN) / 2, PWM_RANGE_MAX, color.v, color2.v);
+				}
+			hOffset = 0;  //this makes throttle function override others, not sure which is best 
```

Comment:
This is wrong. If users configure other functions on the same LED, it means that they **want** these functions to be shown. So simply disabling these functions when 'throttle / aux overlay' is selected is incorrect. (i.e. if they don't want this to happen, they simply will not configure other functions on the same LED.).